awsiotprotocol/loader_test.go

```diff
@@ -16,6 +16,8 @@ package awsiotprotocol
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Personally, I like Golang's testing strategy. https://golang.org/doc/faq#testing_framework

Reply:
OK. I'll update the test cases.